### ISSUE TITLE
`capitalized-comments` - support more ignore instructions (e.g. svelte)

### ISSUE THREAD

**Issue by fregante** [xojs/eslint-config-xo#118]
```js
// svelte-ignore custom_element_props_identifier
// biome-ignore lint/suspicious/noDuplicateProperties
// prettier-ignore
// codespell:ignore
```

Some are covered, but I think it's not practical to keep having to extend this config manually when the format is generally the same: `*-ignore`

Can the regex be made more generic? A "false negative" here isn't really a problem anyway
